aspnetcore/publishing/iis.md

```diff
@@ -112,7 +112,7 @@ services.Configure<IISOptions>(options =>
 
 ### web.config
 
-The *web.config* file configures the ASP.NET Core Module and provides other IIS configuration. Creating, transforming, and publishing *web.config* is handled by `Microsoft.NET.Sdk.Web`, which is included when you set your project's SDK at the top of your project (*.csproj*) file, `<Project Sdk="Microsoft.NET.Sdk.Web">`. To prevent the MSBuild target from transforming your *web.config* file, add the **\<IsTransformWebConfigDisabled>** property to your project file with a setting of `true`:
+The *web.config* file primarily configures the ASP.NET Core Module. It may optionally provide additional IIS configuration settings. Creating, transforming, and publishing *web.config* is handled by the .Net Core Web SDK (`Microsoft.NET.Sdk.Web`). The SDK is configured  at the top of your project file (*.csproj*), i.e. `<Project Sdk="Microsoft.NET.Sdk.Web">`. To prevent the SDK from transforming your *web.config* file, add the **\<IsTransformWebConfigDisabled>** property to your project file with a setting of `true`:
```

Comment:
A few grammar and style nits ...
* .Net :point_right: .NET
* "The SDK is configured ..." :point_right: "The SDK is set ..." (and there is a double-space to remove after the word "configured" right now).
* Remove "i.e." (not allowed in docs) ... you can go with the comma there as before.
* A favor to ask: Could you remove the three instances of the word "your" and replace all three of them with the word "the"? We're in the process of removing/reducing the 2nd person in topics.